src/pages/TagCreator/ProjectRepository.js

```diff
@@ -76,7 +76,7 @@ export const ProjectRepositoryInput = ({
   return (
     <>
       <Grid item style={{ padding: '16px 0px' }} xs={12}>
-        <Typography variant='body1'>Project Repository URL</Typography>
+        <Typography variant='body1'>Github Project Repository URL</Typography>
```

Comment:
I know this is a bit picky but it should be "GitHub"

Reply:
Changed to "GitHub"